modules/internal/ArrayViewReindex.chpl

```diff
@@ -109,7 +109,7 @@ module ArrayViewReindex {
       }
     }
 
-    iter these(param tag: iterKind) ref where tag == iterKind.standalone {
+    iter these(param tag: iterKind) ref where tag == iterKind.standalone && !localeModelHasSublocales {
```

Comment:
What happens if the locale model does have sublocales?  It falls back to leader-follower?  Is there precedent for this (in DefaultRectangular, say?)?